docs/features/startup-jobs.md

```diff
@@ -19,20 +19,26 @@ public class MyStartupJob : IJob
 As with CRON jobs, they must be registered in the `AddNCronJob` method.
 
 ```csharp
-Services.AddNCronJob(options => 
+builder.Services.AddNCronJob(options => 
 {
     options.AddJob<MyStartupJob>()
            .RunAtStartup(); // Configure the job to run at startup
 });
+
+var app = builder.Build();
+// Execute all startup jobs
+await app.UseNCronJobAsync();
+app.Run();
 ```
 
-The `RunAtStartup` method ensures that the job is executed as soon as the application starts. This method is useful for scenarios where certain tasks need to be performed immediately upon application launch.
+The `RunAtStartup` in combination with `UseNCronJobAsync` method ensures that the job is executed as soon as the application starts. This method is useful for scenarios where certain tasks need to be performed immediately upon application launch.
```

Comment:
Digging a little bit more into that, it looks like `RunAtStartup()` can also be leveraged through the `IRuntimeJobRegistry`.

This seems a bit strange, considered that the startup phase has already happened.

Thoughts?

Reply:
Guess I'll have a solution for that

Reply:
Said and done - see last commit

Reply:
Yeah - that was a deliberate choice at the time. I didn’t want to hold two versions of the same builder. But maybe with two interfaces, explicit interface implementation and some generic trickery, that works as well!

good catch